src/main/resources/db/changelog/v.0.5/db.changelog-master.yaml

```diff
@@ -1,7 +1,16 @@
 databaseChangeLog:
-  - include:
+ - include:
```

Comment:
Please, add new changesets to the end of the file to avoid unexpected execution sequence.

Reply:
Okey, i will add this to the end